### PR TITLE
Composer - remove extra (QUICK)

### PR DESCRIPTION
**Pull request by ttodua**
- removed extra packages:
- - `recoil` packages are not needed anymore;
- - `react/http` is already inlcudede in `dependencies`;
- - `clue/buzz-react` - idk, but now it doesn't seem to be used anywhere. if any dependency uses it, it will be installed as sub/dependency by that package. ccxt no longer uses it directly
- - `react/event-loop` is already requirement for `react/async` dependency, so no need to be suggested at all.
- - `clue/http-proxy-react` is not needed to be installed, because we have it in static_dependencies.

- edited manual (was too much obsolete)
- edited dynamic class instantiation (because, without it, linter warned: https://imgsh.net/a/dPAsmoE.png )

### CODE REVIEW

File: php/async/Exchange.php [hunk 1]
```diff
@@ -102,10 +102,11 @@ public function fetch($url, $method = 'GET', $headers = null, $body = null) {
                 $request_browser_options = array( 'tcp' => $proxy, 'dns' => false );
                 $this->set_request_browser($request_browser_options);
             } else if ($socksProxy !== null) {
-                if (!class_exists('\Clue\React\Socks\Client')) {
+                $className = '\\Clue\\React\\Socks\\Client';
+                if (!class_exists($className)) {
                     throw new NotSupported($this->id . ' - to use SOCKS proxy with ccxt, at first you need install module "composer require clue/socks-react"');
                 }
-                $proxy = new \Clue\React\Socks\Client($socksProxy);
+                $proxy = new $className($socksProxy);
```
Comment (carlosmiei) [inline php/async/Exchange.php:109]: @ttodua why this change?

Comment (ttodua) [reply php/async/Exchange.php:109]: as top-post says:
> edited dynamic class instantiation (because, without it, linter warned: https://imgsh.net/a/dPAsmoE.png )

Comment (carlosmiei) [reply php/async/Exchange.php:109]: @ttodua have you tested it? sometimes these updates are tricky

Comment (ttodua) [reply php/async/Exchange.php:109]: yes, responsibly say that I have tested in debugger and it worked same as it was before. you can see here (click "execute") to see those approaches in php: https://onlinephp.io/c/6e5d2

**Comment by carlosmiei** [comment]
LGTM

**Review by carlosmiei — APPROVED** [review]
